bindings/lua/Cargo.toml

```diff
@@ -29,9 +29,26 @@ rust-version.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+
+[features]
+default = ["mlua/lua52"]
+lua54 = ["mlua", "mlua/lua54"]
```

Comment:
mula doesn't allow us to pick different lua version. Please pick up a version first until we have better choice.